package.json

```diff
@@ -59,7 +59,7 @@
       },
       {
         "command": "openshift.project.delete",
-        "title": "Delete",
+        "title": "Delete Project",
```

Comment:
Delete context menu items in OpeShift Application Explorer should stay named as 'Delete' not `Delete ${objectType}`.